libs/portals/admin/form-system/src/components/FormLayout/components/MainColumns.tsx

```
@@ -86,7 +92,7 @@ export const MainContentColumn = () => {
           buttonTextCancel={formatMessage(m.cancel)}
           onConfirm={remove}
           disclosureElement={
-            <DeleteButton onClick={() => { }} label={formatMessage(m.delete)} />
+            <DeleteButton onClick={() => {}} label={formatMessage(m.delete)} />
```

ESLint

[error] Unexpected empty arrow function.

(@typescript-eslint/no-empty-function)